jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh

```diff
@@ -22,5 +35,7 @@ python host_os.py \
 	--push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/${GITHUB_ORGANIZATION_NAME}.github.io.git" \
 	--push-repo-branch "$COMMIT_BRANCH"
 
+create_pr "${GITHUB_ORGANIZATION_NAME}.github.io"
+
 echo "VERSIONS_REPO_COMMIT=$COMMIT_BRANCH" > BUILD_PARAMETERS
 echo "$RELEASE_DATE" > WEEKLY_DIR_NAME
```

Comment:
side-comment not related to this patch: pre_build_script.sh should be entirely in shell script (instead of deferring to a Python script)? what is the current criteria to use Python or shell script in our automation code? In PowerKVM, we used Python for everything, shell script steps in Jenkins only called Python code.